ts/import-log/index.ts

```diff
@@ -33,3 +33,8 @@ export async function setupImportLogPage(
         },
     });
 }
+
+if (window.location.hash.startsWith("#test-")) {
+    const path = window.location.hash.replace("#test-", "");
+    setupImportLogPage({ type: "apkg", path });
+}
```

Comment:
https://github.com/ankitects/anki/assets/180542/c13a4182-784f-4a49-afb1-48776e781ddc

Reply:
I can reproduce this in Firefox too. The following warnings are printed to the console:
![image](https://github.com/ankitects/anki/assets/41397710/47166707-5125-42e5-bb0d-06ca67a28338)

Linked page: https://firefox-source-docs.mozilla.org/performance/scroll-linked_effects.html

Setting `overflow-anchor: none;` on the outer element seems to reduce the jitter, but it doesn't go away.

Reply:
Interesting. I can't seem to detect a change in Safari unfortunately. If I disable the     on:scroll={() => (scrollTop = container.scrollTop)} line, things scroll smoothly, but I reach blank space. I imagine that scrolling is competing with the manual changing of scrollTop. Might be worth looking at how some other virtual tables handle this - my first thought was that maybe we could put an invisible scrollable div on top, which is used to update the scrollTop and thus update the actually-visible elements, without its own scrolling being visible. Not sure if that would work or not though.

Reply:
My implementation is based on https://github.com/Prinzhorn/better-svelte-virtual-list. I used a different approach to make it work with tables. I'll create an issue with some details once this is merged so that me or someone else might investigate it further in the future.